build-gn/generate_vulkan_layers_json.py

```diff
@@ -88,7 +88,8 @@ def main():
     if args.no_path_prefix:
         relative_path_prefix = ''
     elif args.platform == 'Windows':
-        relative_path_prefix = r'..\\'  # json-escaped, hence two backslashes.
+        # https://github.com/KhronosGroup/Vulkan-Tools/issues/866
+        relative_path_prefix = r'.\\'  # json-escaped, hence two backslashes.
```

Comment:
@mikes-lunarg I feel like my fix is a hack since it deviates from the script in VVL. Also it doesn't account for why linux is working which also uses relative paths.

Not sure where I went wrong with the chromium build logic since I really just tried to copy VVL.